modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1987,7 +1987,7 @@ transform.
 
 @param image 8-bit, single-channel binary source image. The image may be modified by the function.
 @param lines Output vector of lines. Each line is represented by a 2 or 3 element vector
-\f$(\rho, \theta)\f$ or \f$(\rho, \theta, \votes)\f$ . \f$\rho\f$ is the distance from the coordinate origin \f$(0,0)\f$ (top-left corner of
+\f$(rho, theta)\f$ or \f$(rho, theta, votes)\f$ . \f$\rho\f$ is the distance from the coordinate origin \f$(0,0)\f$ (top-left corner of
 the image). \f$\theta\f$ is the line rotation angle in radians (
 \f$0 \sim \textrm{vertical line}, \pi/2 \sim \textrm{horizontal line}\f$ ).
 \f$\votes\f$ is the value of accumulator.
```

Comment:
Same thing: `\votes` should be replaced by `votes` or maybe better by `\textrm{votes}`.